schematic/src/g_window.c

```diff
@@ -124,7 +124,7 @@ SCM_DEFINE (current_window, "%current-window", 0, 0, 0,
  * \brief Get the value of the #GschemToplevel fluid.
  * \par Function Description
  * Return the value of the #GschemToplevel fluid in the current dynamic
- * context.
+ * context. Never returns NULL.
```

Comment:
I could even say: "Signals an error if there is no valid window fluid or the fluid value is NULL. Never returns NULL." WDYT?